users/relaying.rst

```diff
@@ -38,6 +61,10 @@ smooth operation::
     Usage of relaysrv:
       -debug
             Enable debug output
+      -ext-address string
+            An optional address to advertising as being available on.
+            Allows listening on an unprivileged port with port forwarding from e.g.                                             
+            443, and be connected to on port 443.
```

Comment:
This is copied from the output of `relaysrv -help`. I was trying to keep it shortish to avoid spamming the console, but agreed that my wording isn't particularly clear.